integrationtest/mapper/orm-spring/pom.xml

```diff
@@ -60,6 +60,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
             <exclusions>
+                <!-- Using a newer version of Hibernate ORM, with a different groupID -->
+                <exclusion>
+                    <groupId>org.hibernate</groupId>
+                    <artifactId>hibernate-core</artifactId>
+                </exclusion>
```

Comment:
This seems unnecessary since we disabled this module and will only re-enable it when switching to Spring Boot 3.